arch/x86/soc/ia32/dts.fixup

```diff
@@ -1,5 +1,7 @@
 /* SoC level DTS fixup file */
 
+#define FLASH_DEV_NAME			FLASH_LABEL
```

Comment:
I would add it in qemu_x86 boards dts.fixup (lest creat it). ia32 might have another native flash driver in the future.